### ISSUE TITLE
Mutt scrolling broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dragonlinux/connectbot#172]
```
What steps will reproduce the problem?
1. Install ConnectBot 1.5.4 from Android Marketplace on a T-Mobile G1
2. SSH to a Linux server with mutt installed
3. Run mutt (in any of the terminal emulation modes)
4. Scroll up in a folder with lots of messages

What is the expected output? What do you see instead?

The scrolling is broken - only the top two lines of the mutt display update
as I scroll up, not the entire list of messages.

What version of the product are you using (you can see this by using Menu
-> About in the Host List)?

1.5.4

What type of system are you trying to connect to?

Linux

If you are able to connect, what is the output of "echo $TERM", "uname -a",
and any other relevant information on the host?

TERM types come through correctly. It's broken with all of the terminal
emulation modes which ConnectBot supports.

Please provide any additional information below.

```

Original issue reported on code.google.com by `richard....@gmail.com` on 24 Jul 2009 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
I can't reproduce this using mutt 1.4.2.2i 

Which version of Mutt are you using?
```

Original comment by `kenny@the-b.org` on 24 Jul 2009 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
Mutt 1.5.19
```

Original comment by `richard....@gmail.com` on 24 Jul 2009 at 10:36

**Comment by GoogleCodeExporter** [comment]
```
I can't reproduce this in Mutt 1.5.17, so I'll get 1.5.19 going later today and 
see
what happens.

Which Linux distribution are you using?
```

Original comment by `kenny@the-b.org` on 24 Jul 2009 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
It's self-compiled, I'm afraid. It's linked against ncurses 5.6.
```

Original comment by `richard....@gmail.com` on 24 Jul 2009 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
I tried this with Mutt 1.5.19 compiled against ncurses 5.7 and couldn't 
reproduce it, 
either... do you have any special settings in muttrc that might be relevant?
```

Original comment by `kenny@the-b.org` on 25 Jul 2009 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
Kenny - I can give you an account on the machine which demonstrates the problem 
if
you want. Please e-mail me privately with your favoured username and password.
```

Original comment by `richard....@gmail.com` on 25 Jul 2009 at 8:57

**Comment by GoogleCodeExporter** [comment]
```
I've sent you details of an account on the machine. It turns out that I needed 
"set
menu_scroll" in the muttrc - perhaps you'll also be able to duplicate elsewhere 
with
that set?
```

Original comment by `richard....@gmail.com` on 29 Jul 2009 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
With menu_scroll set, I can reproduce this issue on any version. I'll take a 
look at it.
```

Original comment by `kenny@the-b.org` on 11 Aug 2009 at 6:01
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I fixed this in r385
```

Original comment by `kenny@the-b.org` on 11 Aug 2009 at 6:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Please can we reopen this - I just upgraded to v1.5.5 (r409) via the 
marketplace. The
behaviour is not fixed.

You can demonstrate on the account that I sent you by logging onto that machine 
and
just scrolling up, as per the original bug description.
```

Original comment by `richard....@gmail.com` on 8 Sep 2009 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I didn't backport this fix to the 1.5 branch. It's still only in the 1.6 
branch.
```

Original comment by `kenny@the-b.org` on 8 Sep 2009 at 1:17
